src/ai/backend/client/cli/admin/user.py

```diff
@@ -410,6 +443,57 @@ def add(
     default=undefined,
     help="Set main access key which works as default.",
 )
+@click.option(
+    "--cuid",
+    "--container-uid",
+    "container_uid",
+    type=OptionalType(int),
+    default=undefined,
+    help="The user ID (UID) that will be assigned to all processes running inside containers created by this user.",
+)
+@click.option(
+    "--unset-cuid",
+    "--unset-container-uid",
+    "unset_container_uid",
+    is_flag=True,
+    default=False,
+    help="Unset the user's container UID.",
+)
+@click.option(
+    "--cgid",
+    "--container-main-gid",
+    "container_main_gid",
+    type=OptionalType(int),
+    default=undefined,
+    help="The primary group ID (GID) that will be assigned to all processes running inside containers created by this user.",
+)
+@click.option(
+    "--unset-cgid",
+    "--unset-container-main-gid",
+    "unset_container_main_gid",
+    is_flag=True,
+    default=False,
+    help="Unset the user's container primary GID.",
+)
+@click.option(
+    "--cgids",
```

Comment:
I feel that abbreviations like `cuid` and `cguid` might overlap with other topics.
How about avoiding using abbreviations for now?